assets/prototype/domain/shared/predicates/tickets/filters/sortTicketsList.ts

```diff
@@ -6,18 +6,18 @@ import { ascend, prop, propOr, sortWith } from 'ramda';
 /**
  * Internal dependencies
  */
-import { SortTickets } from '../../../../eventEditor/data/ticket/types';
+import { SortTicketsBy } from '../../../../eventEditor/data/ticket/types';
 import { Ticket } from '../../../../eventEditor/data/types';
 
-const sortTicketsList = (tickets: Ticket[], order = SortTickets.chronologically) => {
+const sortTicketsList = (tickets: Ticket[], order = SortTicketsBy.chronologically): Ticket[] => {
 	switch (order) {
-		case SortTickets.chronologically:
+		case SortTicketsBy.chronologically:
 			return sortWith([ascend(propOr(null, 'name')), ascend(prop('id'))], tickets);
-		case SortTickets.byName:
+		case SortTicketsBy.byName:
```

Comment:
ye we should have kept the original name here and then changed other code to use `SortTickets.byName` imho

butt I'll create another issue for making that change so that this can move forward